test_utils/utils/setup_fixtures.py

```diff
@@ -0,0 +1,32 @@
+import pathlib
+import json
+
+try:
+	import frappe
+except Exception as e:
+	raise (e)
+
+def get_fixtures_data_from_file(filename):
+	app_dir = pathlib.Path(__file__).resolve().parent.parent / "fixtures"
+	if pathlib.Path.exists(app_dir / filename):
+		with open(app_dir / filename) as f:
+			return json.load(f)
+
+def create_items(settings, shortlist=None):
```

Comment:
@MyuddinKhatri Should the shortlist be connected to some parameter like company? Or is this to be implemented later?

Reply:
@Alchez Following an "explicit is better than implicit" strategy here, so shortlist will install the items with matching PKs. This API should be the same for all the install helpers.

Reply:
@agritheory @Alchez to work with shortlist API should I modify function create_items() - we pass the list of item codes and function creates the items accordingly?

Reply:
@MyuddinKhatri Yes, that what I had in mind. I think the "shortlist" argument name is not great and now is the time to workshop it if we want to make it consistent for all of these functions. Maybe "only_create" is better? It's only a few more characters and is more explicit.

Reply:
@agritheory @Alchez I updated the function now we can pass list of item_codes we want to create from the fixtures. please check.